src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -340,7 +349,22 @@ def get_job_states(self, job_ids: List[str], ts: int = None) -> dict:
         """
         job_ids, error_ids = self._check_job_list(job_ids)
         output_states = self._construct_job_output_state_set(job_ids)
-        return self.add_errors_to_results(output_states, error_ids)
+
```

Comment:
If I had more time I'd think about moving this logic to jobcomm.py ... seems like it might belong there instead

Reply:
I think it belongs better here -- jobcomm.py is a fairly simple wrapper to convert between data structures and messages, whereas the job manager does various manipulations involving job data.